homeassistant/components/shelly/climate.py

```diff
@@ -308,10 +313,6 @@ async def async_added_to_hass(self) -> None:
 
         self.async_on_remove(self.wrapper.async_add_listener(self._update_callback))
```

Comment:
Call super instead, which registers the default update callback.
```suggestion
        await super().async_added_to_hass()
```

Reply:
Done

Reply:
BTW, I used that name to be consistent with the rest of the code in the integration.